src/index.html

```diff
@@ -2,10 +2,14 @@
 <html>
 <head>
     <meta charset="UTF-8"/>
-    <meta name="viewport" content="width=device-width, intitial-scale=1.0"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Onboarding Web Page</title>
 </head>
 <body>
     <h1>Hello World</h1>
+    <button id="apiButton" type="button" >Get Twitter Timeline</button>
+    <div id="apiDataContainer"></div>
+
+    <script type="application/javascript" src="./js/functional.js"></script>
```

Comment:
put this in the head element

Reply:
I put it in the body, as a page load performance thing